### PR TITLE
Start updating zfs docs

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

this change goes together with https://github.com/rancher/os-services/pull/74

### CODE REVIEW

**Comment by joshwget** [comment]
This ZFS service is really different from the old one. We have to make sure to explain that mounts are done outside of the console and that all commands are actually run in a ZFS tools container.

**Comment by SvenDowideit** [comment]
agreed.

File: docs/os/system-services/custom-system-services/index.md [hunk 1]
```diff
@@ -57,6 +57,53 @@ The config settings to set the url in which `ros` should look for an `index.yml`
 
 For example, in RancherOS v0.7.0, the `core` repository is set to `https://raw.githubusercontent.com/rancher/os-services/v0.7.0`.
 
+### Service development and testing
```
Comment (joshwget) [inline docs/os/system-services/custom-system-services/index.md]: Can we make this a separate section on the sidebar?

Comment (SvenDowideit) [inline docs/os/system-services/custom-system-services/index.md]: I think I'll defer that to when we refactor the structure

**Pull request merged**